### PR TITLE
Karpovich: lesson04

### PR DESCRIPTION
**Pull request by sablayz**
close: #15

### CODE REVIEW

File: karpovich-v/lesson04/src/main/webapp/index.jsp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<html>
+<body>
+<h1>Karpovich Vladislav</h1>
+
+<p>Java Back-end</p>
+
+</body>
```
Comment (rakovets) [inline karpovich-v/lesson04/src/main/webapp/index.jsp:7]: You need to add more content

**Review by rakovets — CHANGES_REQUESTED** [review]
You need to add more content

File: karpovich-v/lesson04/src/main/webapp/index.jsp (hunk 1 above)
Comment (sablayz) [reply karpovich-v/lesson04/src/main/webapp/index.jsp]: done

**Review by rakovets — APPROVED** [review]
Approved

**Pull request merged**